package.json

```diff
@@ -24,6 +18,22 @@
     "solc": "^0.7.1",
```

Comment:
I believe this is the most up-to-date version of the solc compiler. Unfortunately, the Solidity code for our contracts is written in a version that is no longer supported by this compiler. In order to get things working well have to use version `^0.4.25` of the solc compiler and hopefully transition to the newer versions as we move forward.